### DISCUSSION TITLE
Some Font Awesome icons not rendering

### DISCUSSION THREAD

**Discussion by aforsythe** [squidfunk/mkdocs-material#3891] [Q&A]
Not sure what happened ... this was previously working fine for me ... 

I'm now only getting some font awesome icons to render in markdown with the latest version of the docker container.

<img width="491" alt="Screen Shot 2022-05-06 at 10 51 55 AM" src="https://user-images.githubusercontent.com/309001/167190907-64a0cb95-15cb-4a5d-b34c-155420c268c2.png">

[`mkdocs.yml`](https://github.com/ampas/aces-docs/blob/dev/mkdocs.yml) is 

```
  - pymdownx.emoji:
      emoji_index: !!python/name:materialx.emoji.twemoji
      emoji_generator: !!python/name:materialx.emoji.to_svg
```

Any hints would be greatly appreciated

**Comment by facelessuser** [comment]
If they used to work, but are now not rendering, it is likely the names changed in some recent revision. Those names are not controlled by Material.

**Reply by aforsythe** [reply]
Understood ... but the names seem to be right.

https://github.com/squidfunk/mkdocs-material/blob/master/material/.icons/fontawesome/solid/arrow-down-wide-short.svg

**Reply by facelessuser** [reply]
Okay, so when I wrote the Material emoji extension, I generalized it. So I look in Material's `.icons` folder. After that, the icon names are found as long as you follow whatever the director/filename structure is. For the icon you listed, it would be:

```
:fontawesome-solid-arrow-down-wide-short:
```

Makes sense?

**Reply by aforsythe** [reply]
I'm not following ... that's exactly what I have.

**Reply by aforsythe** [reply]
https://github.com/ampas/aces-docs/blob/dev/mkdocs/docs/index.md

**Reply by facelessuser** [reply]
> I'm not following ... that's exactly what I have.

I apologize, I missed that.

All I can say is that it is working for me, even using your syntax as below it still works:

```
:fontawesome-solid-arrow-down-wide-short:{.icons}
```

<img width="247" alt="Screen Shot 2022-05-06 at 12 55 11 PM" src="https://user-images.githubusercontent.com/1055125/167201080-affaaead-5650-4747-ac2d-5848d0e58f6b.png">

I suspect that it may be related to some plugin interfering, or something like that. I am also using the very latest public Material version.

**Reply by aforsythe** [reply]
Humm. Ok. That's weird.  Like I said it was working before and it appears to not work with my old docker image or a new one based on the latest mkdocs-material release.  I'll dig in further.

**Comment by snim2** [comment]
I have exactly the same problem, for me `:fontawesome-solid-random:`, `:fontawesome-solid-drafting-compass: `, and `:fontawesome-solid-hands-helping:` no longer work. Bumping `mkdocs-material` to the latest version doesn't seem to help.

**Reply by snim2** [reply]
But I see that `"fontawesome/solid/compass-drafting"` has maybe been named slightly differently to the [original FA icon](https://fontawesome.com/v5/icons/drafting-compass?style=solid&s=solid).

**Reply by snim2** [reply]
OK, so in my case, I think it's a combination of that, and FA being updated in `mkdocs-material` and some of the icons changing name...

**Reply by facelessuser** [reply]
Yep, FA, at any time, may change names. If that happens, you'll have to update names accordingly.
